### DISCUSSION TITLE
Embed superset report in external web application and control the filters externally.

### DISCUSSION THREAD

**Discussion by aravpal** [apache/superset#19034] [Q&A / Help]
I am trying to embed superset report in external web SPA application in Angular. I am able to render the report in IFrame, now my question is how to control the filters externally from web application? I have to customize filters using my own GUI component in Angular. Is it possible to change the filters and render the data in Iframe?

**Comment by TebaleloS** [comment]
Can you please share the solution, I am trying to do the same thing but I am failing.

**Comment by pratikpparikh** [comment]
@aravpal can you share a sample of how you are embedding the superset in iFrame.

**Comment by aravpal** [comment]
Sorry, I am no longer working on this project and don’t have any samples
handy at the moment.
